### PR TITLE
count form submission types

### PR DESCRIPTION
**Pull request by benrudolph**
@snopoke couldn't really think of any other useful tags to include, let me know if you have any ideas. ran this with a few regular submissions

buddy: @calellowitz

### CODE REVIEW

**Review by snopoke — APPROVED** [review]

**Comment by snopoke** [comment]
pitty we can't get it in a single metric with tags

**Comment by benrudolph** [comment]
actually we almost can, just one early return statement

**Comment by benrudolph** [comment]
that might be better, will update

File: corehq/form_processor/submission_post.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
     CaseValueError
 from casexml.apps.case.xml import V2
 from corehq.toggles import ASYNC_RESTORE
+from corehq.util.datadog.gauges import datadog_counter
```
Comment (dimagimon) [inline corehq/form_processor/submission_post.py]: F401 'datadog_counter' imported but unused

**Comment by benrudolph** [comment]
added `submission_type` to the FormProcessingResult

**Review by snopoke — APPROVED** [review]

**Review by snopoke — APPROVED** [review]

**Pull request merged**